build.gradle

```diff
@@ -118,197 +125,197 @@ configurations {
 
-    compile "com.github.vagmcs:optimus_${scalaBinaryVersion}:3.1.0"
-    compile "com.github.vagmcs:optimus-solver-oj_${scalaBinaryVersion}:3.1.0"
+    implementation "com.github.vagmcs:optimus_${scalaBinaryVersion}:3.1.0"
+    implementation "com.github.vagmcs:optimus-solver-oj_${scalaBinaryVersion}:3.1.0"
 
-    compile group: 'io.spray', name: 'spray-json_2.12', version: '1.3.5'
+    implementation group: 'io.spray', name: 'spray-json_2.12', version: '1.3.5'
 
     ////////////////////////////////////
     ///Performance Monitoring (Kamon)///
     ////////////////////////////////////
 
     // compile("io.kamon:kamon-bundle_${scalaBinaryVersion}:${kamonVersion}")
-    compile("io.kamon:kamon-core_${scalaBinaryVersion}:2.0.1")
+    implementation("io.kamon:kamon-core_${scalaBinaryVersion}:2.0.1")
     // compile("io.kamon:kamon-influxdb_2.12:2.0.0")
 
-    compile("org.influxdb:influxdb-java:2.16")
+    implementation("org.influxdb:influxdb-java:2.16")
 
     /////////////
     // Akka Dependencies
     ////////////
 
     // CORE Akka //
-    compile group: 'com.typesafe.akka', name: "akka-actor_${scalaBinaryVersion}", version: akkaBinaryVersion
-    compile group: 'com.typesafe.akka', name: "akka-slf4j_${scalaBinaryVersion}", version: akkaBinaryVersion
-    compile group: 'com.typesafe.akka', name: "akka-cluster_${scalaBinaryVersion}", version: akkaBinaryVersion
-    compile group: 'com.typesafe.akka', name: "akka-cluster-tools_${scalaBinaryVersion}", version: akkaBinaryVersion
+    implementation group: 'com.typesafe.akka', name: "akka-actor_${scalaBinaryVersion}", version: akkaBinaryVersion
+    implementation group: 'com.typesafe.akka', name: "akka-slf4j_${scalaBinaryVersion}", version: akkaBinaryVersion
+    implementation group: 'com.typesafe.akka', name: "akka-cluster_${scalaBinaryVersion}", version: akkaBinaryVersion
+    implementation group: 'com.typesafe.akka', name: "akka-cluster-tools_${scalaBinaryVersion}", version: akkaBinaryVersion
 
-    compile group: 'com.typesafe.akka', name: "akka-http_${scalaBinaryVersion}", version: "10.1.12"
-    compile group: 'de.heikoseeberger', name: "akka-http-circe_${scalaBinaryVersion}", version: "1.25.2"
+    implementation group: 'com.typesafe.akka', name: "akka-http_${scalaBinaryVersion}", version: "10.1.12"
+    implementation group: 'de.heikoseeberger', name: "akka-http-circe_${scalaBinaryVersion}", version: "1.25.2"
 
     // TEST Akka //
-    testCompile group: 'com.typesafe.akka', name: "akka-testkit_${scalaBinaryVersion}", version: akkaBinaryVersion
+    testImplementation group: 'com.typesafe.akka', name: "akka-testkit_${scalaBinaryVersion}", version: akkaBinaryVersion
 
-    scoverage "org.scoverage:scalac-scoverage-plugin_${scalaBinaryVersion}:1.3.1", "org.scoverage:scalac-scoverage-runtime_${scalaBinaryVersion}:1.3.1"
+//    scoverage "org.scoverage:scalac-scoverage-plugin_${scalaBinaryVersion}:1.4.8", "org.scoverage:scalac-scoverage-runtime_${scalaBinaryVersion}:1.4.8"
```

Comment:
Was this commented for testing? Otherwise why remove?

Reply:
@JustinPihony it was just testing